Library/Homebrew/cmd/deps.rb

```diff
@@ -66,6 +66,10 @@ class Deps < AbstractCommand
                             "debugging the `--installed`/`--eval-all` display mode."
         switch "--HEAD",
                description: "Show dependencies for HEAD version instead of stable version."
+        flag   "--os=",
+               description: "Show dependencies for the given operating system."
+        flag   "--arch=",
+               description: "Show dependencies for the given CPU architecture."
```

Comment:
Tangential: Do we list supported operating systems and architectures in the docs anywhere? If not, that might be a good thing to add at some point since this pattern is showing up in a few commands.

Reply:
I don't think it is documented. Closest may be `on_*` docs - https://docs.brew.sh/Formula-Cookbook#handling-different-system-configurations

Supported architectures at least show up if unsupported one is passed in, e.g.
```
Error: New arch must be :arm or :intel
```